RecoTracker/MkFitCore/src/Config.cc

```diff
@@ -0,0 +1,121 @@
+#include "RecoTracker/MkFitCore/interface/Config.h"
+
+#include "RecoTracker/MkFitCore/interface/TrackerInfo.h"
+#include "RecoTracker/MkFitCore/interface/IterationConfig.h"
+
+namespace mkfit {
+
+  namespace Config {
+    TrackerInfo TrkInfo;
+    IterationsInfo ItrInfo;
+
+    int nTracks = 10000;
+    int nEvents = 20;
+    int nItersCMSSW = 0;
+    bool loopOverFile = false;
+
+    int nTotalLayers = -1;
+
+    std::string geomPlugin = "CylCowWLids";
+
+    // Multi threading and Clone engine configuration
+    int numThreadsFinder = 1;
+    int numThreadsEvents = 1;
+
+#if defined(__AVX512F__)
+    int numThreadsSimulation = 60;
+#else
+    int numThreadsSimulation = 12;
+#endif
+
+    int finderReportBestOutOfN = 1;
+
+    /*MM: moving out to IterationParams*/
+    //int   nlayers_per_seed = 3; // can be overriden from Geom plugin; a very confusing variable :)
+    int numSeedsPerTask = 32;
+
+    // number of hits per task for finding seeds
+    int numHitsPerTask = 32;
+
+    // material effects
+    float RlgridME[Config::nBinsZME][Config::nBinsRME];
+    float XigridME[Config::nBinsZME][Config::nBinsRME];
+
+    //float chi2Cut = 15.;
+    //float chi2CutOverlap = 5.;
+    //float pTCutOverlap = 0.;
```

Comment:
remove

Reply:
Addressed in Round 1 follow-up PR trackreco/cmssw#75 commit: https://github.com/trackreco/cmssw/pull/75/commits/9da0fa99e0728f40a2a74e9fd969900b9de96849